content/Foundation/Summit/2022/home.pml

```diff
@@ -53,7 +53,7 @@ Munich, Germany (hybrid), 10-13 Oct 2022
     Location & Host
   </h3>
   <p>
-    The seL4 Summit 2022 was held in Munich, Germany (hybrid), 10-13 Oct 2022.
+    The <a href="https://hensoldt-cyber.com/sel4-summit-2022/">seL4 Summit 2022</a> was held in Munich, Germany (hybrid), 10-13 Oct 2022.
```

Comment:
I suggest:

The seL4 Summit 2022 was <a href="https://hensoldt-cyber.com/sel4-summit-2022/">held in Munich, Germany (hybrid), 10-13 Oct 2022, hosted by HENSOLDT Cyber GmbH</a>.

And remove the paragraph below. It might make the section very small but I think that's ok.